1-exercises/F-for-of-loop/exercise.js

```diff
@@ -4,13 +4,19 @@
 
 // TODO Use a for-of loop to output each of the tube stations below.
 let tubeStations = [
-    "Aldgate",
-    "Baker Street",
-    "Picadilly Circus",
-    "Oxford Street",
-    "Tottenham Court Road"
+  "Aldgate",
+  "Baker Street",
+  "Picadilly Circus",
+  "Oxford Street",
+  "Tottenham Court Road",
 ];
 
+for (let stations of tubeStations) {
+  console.log(stations);
+}
 
 // TODO Use a for-of loop to capitalise and output each letter in the string seperately.
 let str = "codeyourfuture";
+for (let letters of str) {
```

Comment:
The variable letters would read better if it was named letter.  You're grabbing a single letter in a group of letters and then doing something with that letter inside the for loop statement body before returning it.

Reply:
Corrected.